index.js

```diff
@@ -20,7 +20,7 @@ cache.setup(() => {
   artsyXapp.init({ url: API_URL, id: CLIENT_ID, secret: CLIENT_SECRET }, () => {
     // Start the server
     if (module === require.main) {
-      app.listen(PORT, () => console.log(`Force listening on port ${PORT}`))
+      app.listen(PORT, () => console.log(`\n\n  [Force] Booting on port ${PORT}... \n`))
```

Comment:
If you change this to mention `http://localhost:${PORT}` then the URL will be command+clickable in macOS terminals ;)

Reply:
Oh yeah, this is deliberate so that the "compile" and "launch" phases remain distinct and the user doesn't open a browser to broken page. Once webpack is finished it outputs a clickable link 👍

Reply:
Ah kk 👍